documentation/source/contribute/contribute_openebs_doc.rst

```diff
@@ -0,0 +1,73 @@
+
+Contributing to the OpenEBS Documentation
+=========================================
+
+You can access the latest documents at http://openebs.readthedocs.io/en/latest/. 
+
+Automated builds are setup for OpenEBS documentation at https://readthedocs.org/projects/openebs/
+
+Sphinx is used for building the OpenEBS documentation - http://www.sphinx-doc.org/en/stable/tutorial.html
+
+Documentation Contributors
+--------------------------
+
+1. If you would like to give feedback on existing content, create an issue on documentation (see, `Creating an Issue`_).
+
+2. If you would like to contribute to new content, 
+     -  create an issue (see, `Creating an Issue`_), 
+     -  create your own branch (see, `Creating a Branch`_), 
+     -  work on the content by creating an RST file, and 
+     -  create a pull request (see, `Creating a Pull Request`_).
+
+Internal Contributors
```

Comment:
The section should be renamed as - Review Process for Documentation Issues and Pull Requests

Reply:
Done